test/addon_product_test.rb

```diff
@@ -49,7 +61,8 @@
           ProductFactory.create_product("kind" => :product,
            "name" => "new_product", "version" => "1.0",
            "arch" => "x86_64", "source" => "1",
-           "product_package" => "new_product-release")
+           "product_package" => "new_product-release",
+           "deps" => [])
```

Comment:
I wonder whether it is possible that "deps" is missing. In that case, `Y2Packager::Resolvable#method_missing` will raise an exception.

```
NoMethodError:
       undefined method `deps' for #<Y2Packager::Resolvable:0x000055b04eb50848>
```

Reply:
Yes, a good point.